src/App.tsx

```diff
@@ -416,7 +356,7 @@ function AppContent() {
             </Route>
             <Route exact path={APP_PATHS.SPECIES}>
               {selectedOrgHasSpecies() ? (
-                <SpeciesList reloadData={reloadSpecies} species={species} />
+                <SpeciesList reloadData={reloadSpecies} species={species || []} />
```

Comment:
maybe extract out into a `const nonNullSpecies = useMemo<Species[]>(() => species || [], [species]);` since we use this in multiple places.

Reply:
In my opinion it doesn't make sense to force the check and jam into React's memory since it isn't an "expensive operation"